src/DiskIO/IpcIo/IpcIoFile.cc

```diff
@@ -525,6 +533,20 @@ IpcIoFile::HandleNotification(const Ipc::TypedMsgHdr &msg)
         HandleResponses("after notification");
 }
 
+/// Handle incoming messages in IPC queue without relying on the queue state
+/// being in sync with our state (because the queue may be reflecting the state
+/// of our killed predecessor).
```

Comment:
Still needs the same fear removal that CollapsedForwarding::HandleNewDataAtStart() had.

```suggestion
/// Handle queued IPC messages for the first time in this process lifetime, when
/// the queue may be reflecting the state of our killed predecessor.
```
or, being identical text and we could redux one of them to just use the \sa connection.

```suggestion
```

Reply:
Done. For clarity, I decided to make a copy of the description here. I guess we could use \copydoc in this case, but that would render \sa useless.

Reply:
I am working on a way to use `\copydoc`.